networking/kvm.go

```diff
@@ -119,6 +120,11 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf) (netlink.Link,
 	if err := netlink.LinkAdd(link); err != nil {
 		return nil, errwrap.Wrap(errors.New("cannot create macvtap interface"), err)
 	}
+	if err := netlink.LinkSetUp(link); err != nil {
+		// remove newly added link ignoring errors, we are already in errornous situation
```

Comment:
remove the newly added link and ignore errors, because we already are in a failed state